install/linux/usr/share/odemis/sim/sparc2-ded-focus-test-sim.odm.yaml

```diff
@@ -0,0 +1,337 @@
+}
+
+# Must be connected on AI 0/AI GND
+"SEM Detector": { # aka ETD
+    # Internal child of SEM Scan Interface, so no class
+    role: se-detector,
+    init: {
+        channel: 0, # 0-> sawtooth waves, 1-> square waves
+        limits: [-3, 3] # V
+    },
+}
+
+# In reality, this is a Zyla, but you need libandor3-dev to simulate an AndorCam3
+# Depending exactly on the configuration, it might also be used for spectrometer
+"Camera": {
+    class: simcam.Camera,
+    role: ccd,
+    init: {image: "sparc-spec-sim.h5",
+    },
+}
+
+
+# Spectrometer 1 = Andor Zyla + Shamrock SR193 (connected via USB)
+"Integrated Spectrometer": {
+    class: spectrometer.CompositedSpectrometer,
+    role: spectrometer-integrated,
+    children: {detector: "Camera", spectrograph: "Integrated Spectrograph"},
+    init: {
+        transp: [1, 2], # only applied to the spectrometer data (not raw CCD)
+    },
+    properties: {
+        binning: [4, 16],
+    },
+}
+
+"Integrated Spectrograph": {
+    class: andorshrk.Shamrock,
+    role: spectrograph,
+    power_supplier: "Power Control Unit",
+    init: {
+        device: "fake",
+        slits: {1: "slit-in", # INPUT_SIDE
+        },
+        bands: { # The filters in the filter-wheel
+            6: "pass-through"
+        },
+        fstepsize: 10.9e-6, # m, size of one focus step
+    },
+    affects: ["Camera", "Integrated Spectrometer"],
+}
+
+# Provide the mirror (aka port selector) of the spectrograph as a separate component
+# Only used to disable the AR CCD, to perform basic background subtraction
+"Spec CCD Flipper": {
+    class: actuator.FixedPositionsActuator,
+    role: "spec-det-selector",
+    children: {"rx": "Integrated Spectrograph"},
+    init: {
+        axis_name: "flip-out",
+        positions: {
+            0: ["Camera", "Integrated Spectrometer"],
+            1.5707963267948966: [],   # nothing connected
+        },
+    },
+    affects: ["Camera"],
+}
+
+
+# Spectrometer = Andor iDus + Shamrock SR303i (connected via USB)
+# Note that the SR193 could be used as a spectrometer too, but it's not needed, and that keeps things simple
+"Dedicated Spectrometer": {
+    class: spectrometer.CompositedSpectrometer,
+    role: spectrometer,
+    children: {detector: "Spectral Camera", spectrograph: "External Spectrograph"},
+    properties: {
+        binning: [4, 16],
+    },
+}
+
+"External Spectrograph": {
+    class: andorshrk.Shamrock,
+    role: spectrograph-dedicated,
+    children: {"ccd": "Spectral Camera"},
+    init: {
+        device: "fake",
+        slits: {1: "slit-in", # INPUT_SIDE
+        },
+    },
+    affects: ["Spectral Camera", "Dedicated Spectrometer"],
+}
+
+# Dedicated camera for the acquisition of the spectra
+"Spectral Camera": {
+    class: simcam.Camera,
+    role: sp-ccd,
+    init: {image: "sparc-spec-sim.h5",
+    },
+    children: {focus: "Spectrograph focus"}
```

Comment:
Sorry, I've just changed the simcam, and now it uses a separate focus actuator, instead of providing its own. That means that you could create a simulator with both cameras being a simcam, and depending on the same focus actuator. In turns, this means that probably you should be able to make do with a single simulator file. 

I'd suggest that you rebase your branch with the master, and then check the changes on `linux/usr/share/odemis/sim/sparc2-focus-test.odm.yaml` (see commit 64bc83d87ee3).